### ISSUE TITLE
findutils@4.4.2: hash check failed

### ISSUE THREAD

**Issue by s4ik4t** [ScoopInstaller/Main#818]
PS C:\WINDOWS\system32> scoop install findutils
Installing 'findutils' (4.4.2) [64bit]
msysCORE-1.0.13-2-msys-1.0.13-bin.tar.lzma (1.4 KB) [====================================================================================================================================] 100%
Checking hash of msysCORE-1.0.13-2-msys-1.0.13-bin.tar.lzma ... ERROR Hash check failed!
App:         main/findutils
URL:         https://downloads.sourceforge.net/project/mingw/MSYS/Base/msys-core/msys-1.0.13-2/msysCORE-1.0.13-2-msys-1.0.13-bin.tar.lzma
First bytes: 3C 68 74 6D 6C 3E 0D 0A
Expected:    fff020614bd796697739887c636f15a42b4b14aa
Actual:      6cc7a5b8664490ab285116a3117752e0929a0ab9
SourceForge.net is known for causing hash validation fails. Please try again before opening a ticket.

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=findutils%404.4.2%3a+hash+check+failed

**Comment by Ash258** [comment]
Based on the size of downloaded file, the URL is blocked. If you open the file in text editor you will most likely see html with message it is blocked by firewall or anything

![image](https://user-images.githubusercontent.com/13260377/74710400-da6a0100-5221-11ea-9978-a8d21d521e82.png)
